src/data/directory/lgu/region-i-ilocos-region.json

```
@@ -1071,18 +1062,17 @@
             "contact": "(075) 523-6168"
           },
           "vice_mayor": {
-            "name": "MARK STEPHEN D. MEJIA",
-            "contact": "523-9624"
+            "name": "JOHNNY CABRERA"
```

```suggestion
            "name": "FERNANDO JUAN 'JOHNNY' A. CABRERA"
```

ref: https://lgu201.dilg.gov.ph/view.php?r=01&p=055&m=26